### DISCUSSION TITLE
Python support removed for Axis OS 12.0 ?

### DISCUSSION THREAD

**Discussion by egSat** [AxisCommunications/acap-computer-vision-sdk-examples#197] [General] [answered]
Hi,
From https://github.com/AxisCommunications/acap-computer-vision-sdk we can read that computer vision sdk is deprecated and starting at Axis OS 12 only the native sdk will be used. 
That means that also the possibility of running Python with OpenCV inside containers will be removed?

thank you very much

**Comment by pataxis** [comment, marked as answer]
Hi @egSat , 

as stated in the banner

- `aarch64` will be supported until the end of the AXIS OS 12 LTS cycle.
- CV SDK is removed in ACAP release 12.0, functionality has moved to [acap-runtime](https://github.com/AxisCommunications/acap-runtime) instead.

It will be possible to run container applications on `aarch64` products with container support during AXIS OS 12 releases.
But note that the deprecation means that it's discouraged to use container applications in production.
After AXIS OS 13.0 the current solution for container support will be removed.

More information will come in https://axiscommunications.github.io/acap-documentation/ and we'll see if can make the message clearer.

**Comment by sebaxis** [comment]
Hi.

[All products with existing container support](https://www.axis.com/support/tools/product-selector/shared/%5B%7B%22index%22%3A%5B10%2C2%5D%2C%22value%22%3A%22Yes%22%7D%5D) will be supported until 2032 when 2026 LTS reaches end of life.

So you can still build container based applications running python and using OpenCV.
